onboarding-checklist.md

```diff
@@ -0,0 +1,145 @@
+# Onboarding Checklist #
+
+## Instructions ##
+
+When someone new joins the team:
+
+1. A team member adds the new team member to the `cisagov` organization in
+GitHub.
+1. Create a new issue called "Onboard [NewPerson]".
+1. View the raw source of this file.
+1. Copy the text after the line to the issue, replacing "NewPerson" with the
+new person's name and "Buddy" with the onboarding buddy's name.
+1. Remove any block of tasks that doesn't apply to the new team member's role
+1. Submit the issue.
+1. Assign the new person and the new person's Onboarding Buddy to the issue.
+1. Move the issue into _In Progress_.
+
+---
+
+To get NewPerson productively contributing to the team, Buddy should help
+NewPerson complete a prescribed set of tasks that will bring them up to speed.
+
+## Directions ##
+
+**NewPerson and Buddy:** Try to go through your checklists in order.
+
+**Buddy:** If you can’t complete any of the items on your checklist
+personally, _you are responsible for ensuring that someone with the correct_
+_access completes that item_.
```

Comment:
I don't think the trailing `_` after "correct" or the leading one before "access" are necessary.

Reply:
Artifact of my editor - it didn't italicize across the line break. Resolved in 802d2dd2215abd0eda70209fe993f0b5d8227ad4

Reply:
I think we should keep the italics, I was just pointing out that they don't have to be stopped and started across a line break.

Reply:
Yep, the italics stayed! the extra underscores just made my editor display the italicization.

Reply:
No, the italics have been removed.

Reply:
Italics reanimated in commit a90236a.